setup.go

```diff
@@ -72,8 +72,6 @@ func setup(c *caddy.Controller) error {
 func browseLoop(m *MDNS) {
 	for {
 		m.BrowseMDNS()
-		// 5 seconds seems to be the minimum ttl that the cache plugin will allow
-		// Since each browse operation takes around 2 seconds, this should be fine
-		time.Sleep(5 * time.Second)
+		time.Sleep(60 * time.Second)
```

Comment:
LGTM, but I have one Q,
Do we know what processes (besides reducing the mcast traffic ) this change may affect?  
maybe add-worker node?

Reply:
I'm not sure it would affect anything at this point. It turns out we don't need the node records at all.

I think I'm just going to close this PR since we're planning to remove the mdns bits entirely.